frontend/src/utils/account-send.ts

```diff
@@ -0,0 +1,89 @@
+import { encrypt, decrypt } from 'ethereum-cryptography/aes';
+import { hexToBytes, utf8ToBytes, bytesToUtf8 } from 'ethereum-cryptography/utils';
+import { getRandomBytesSync } from 'ethereum-cryptography/random';
+import localforage from 'localforage';
```

Comment:
Thoughts on using the [config](https://github.com/localForage/localForage#configuration) to avoid using Web SQL here? Seems we may be able to force using indexed DB, since Web SQL is being [deprecated](https://developer.chrome.com/blog/deprecating-web-sql/)

Reply:
Totally agree